### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TactualNick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ Ambitious digital identification system
...
-## Configure direnv
-Installing direnv will allow the python deps to be configured when entering
-and exiting specific folders in the repo.
-
-## Install direnv
+## Install dependencies
 ```
-$ sudo apt-get install direnv
+$ sudo apt-get update
+$ sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git direnv
```
Comment (mikelaferriere) [inline README.md:10]: oh yea, I definitely probably already had a ton of these already installed...

File: README.md [hunk 2]
```diff
@@ -39,14 +39,16 @@ $ echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\
...
 ### Restart shell
+If using WSL the following won't work. Close and reopen the WSL window instead.
 ```
 $ exec "$SHELL"
 ```
 
-### Install proper python version
+### Install proper python versions
 ```
+$ pyenv install 2.7.15
```
Comment (TactualNick) [inline README.md]: fyi this was to get python 2 installed in WSL. Figured it was okay here, but could see the argument for pulling it out to a WSL only section

Comment (mikelaferriere) [inline README.md]: what do we need python2 for?

**Review by mikelaferriere — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mikelaferriere) [reply README.md]: I see, maybe just add a note about WSL? I think its fine here tho

Comment (TactualNick) [inline README.md]:
```suggestion
$ pyenv install 2.7.15 # WSL only
```

**Review by mikelaferriere — APPROVED** [review]

**Pull request merged**